ansible_collection/hpe/nimble/plugins/module_utils/hpe_nimble.py

```diff
@@ -43,21 +42,6 @@ def remove_null_args(**kwargs):
     return kwargs
 
 
-def is_dict_key_value_present_in_server_dict(server_dict, dict_to_check):
```

Comment:
this is the same[https://github.com/hpe-storage/nimble-ansible-modules/pull/1#discussion_r443311462]( https://github.com/hpe-storage/nimble-ansible-modules/pull/1#discussion_r443311462)